src/AspNetCoreModuleV2/CommonLib/FileOutputManager.cpp

```diff
@@ -32,132 +35,91 @@ FileOutputManager::Initialize(PCWSTR pwzStdOutLogFileName, PCWSTR pwzApplication
     return S_OK;
 }
 
-bool FileOutputManager::GetStdOutContent(STRA* struStdOutput)
-{
-    //
-    // Ungraceful shutdown, try to log an error message.
-    // This will be a common place for errors as it means the hostfxr_main returned
-    // or there was an exception.
-    //
-    CHAR            pzFileContents[4096] = { 0 };
-    DWORD           dwNumBytesRead;
-    LARGE_INTEGER   li = { 0 };
-    BOOL            fLogged = FALSE;
-    DWORD           dwFilePointer = 0;
-
-    if (m_hLogFileHandle != INVALID_HANDLE_VALUE)
-    {
-        if (GetFileSizeEx(m_hLogFileHandle, &li) && li.LowPart > 0 && li.HighPart == 0)
-        {
-            if (li.LowPart > 4096)
-            {
-                dwFilePointer = SetFilePointer(m_hLogFileHandle, -4096, NULL, FILE_END);
-            }
-            else
-            {
-                dwFilePointer = SetFilePointer(m_hLogFileHandle, 0, NULL, FILE_BEGIN);
-            }
-
-            if (dwFilePointer != INVALID_SET_FILE_POINTER)
-            {
-                // Read file fails.
-                if (ReadFile(m_hLogFileHandle, pzFileContents, 4096, &dwNumBytesRead, NULL))
-                {
-                    if (SUCCEEDED(struStdOutput->Copy(pzFileContents, dwNumBytesRead)))
-                    {
-                        fLogged = TRUE;
-                    }
-                }
-            }
-        }
-    }
-
-    return fLogged;
-}
-
+// Start redirecting stdout and stderr into the file handle.
+// Uses sttimer to continuously flush output into the file.
 HRESULT
 FileOutputManager::Start()
 {
     SYSTEMTIME systemTime;
     SECURITY_ATTRIBUTES saAttr = { 0 };
     STRU struPath;
 
+    // Concatenate the log file name and application path
     RETURN_IF_FAILED(FILE_UTILITY::ConvertPathToFullPath(
         m_wsStdOutLogFileName.QueryStr(),
         m_wsApplicationPath.QueryStr(),
         &struPath));
 
     RETURN_IF_FAILED(FILE_UTILITY::EnsureDirectoryPathExist(struPath.QueryStr()));
 
+    // Get the module name and add it to the log file name
+    // This change was made 
```

Comment:
??

Reply:
Not a fan of having dll name in log file name.

Reply:
Do we care about having unique log files between the shim and request handler? Or the same file?

Reply:
Same file is better.

Reply:
We'll need to pass the file name/ check if a file is already present in the directory?

Reply:
Or we can use https://docs.microsoft.com/en-us/windows/desktop/api/processthreadsapi/nf-processthreadsapi-getprocesstimes.

I'm fine having additional `iisexpress.exe\w3wp.exe` log get_native_directories call but having two logs for the same inprocess process is strange.